src/components/layout/Footer.astro

```diff
@@ -30,7 +30,7 @@ const navLinks = [
           <a
             href={link.href}
             target={link.target ? link.target : "_self"}
-            class="text-sm leading-6 text-gray-100 hover:text-white"
+            class="flex items-center gap-1.5 text-sm leading-6 text-gray-100 hover:text-white"
```

Comment:
Small nitpick for centering logos and text in the footer. I am also remove it from this PR if you would prefer.